lblod-decision-service/endpoints/decision/decision.controller.js

```diff
@@ -31,17 +31,14 @@ const getAll = async (req, res, next) => {
 };
 
 const validate = async (req, res, next) => {
-  const { id, hash, signedResourceId } = JSON.parse(req.body.validation);
+  const { id, hash } = JSON.parse(req.body.validation);
   const result = await decisionService.Validate(id, hash);
```

Comment:
```suggestion
   const { result, blockchainHash } = await decisionService.Validate(id, hash);
```
En dan lijn 39, 40 hoef je ze niet 2x ge schrijven